src/index.test.ts

```diff
@@ -192,7 +192,7 @@ describe('Running @erc725/erc725.js tests...', () => {
         web3.currentProvider,
       );
       const result = await erc725.getData();
-      assert.deepStrictEqual(result, e2eResults);
+      assert.notDeepStrictEqual(result, e2eResults);
```

Comment:
why switch from toEqual to **not** equal ??

Reply:
@Hugoo understood the issue.
I have reverted the changes to use the same schema for the e2e tests.